### PR TITLE
GHC-95: Configure patient dashboard

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/openmrs/apps/clinical/dashboard.json [hunk 1]
```diff
@@ -172,23 +194,133 @@
...
+      },
+      "socialEconomic": {
+        "translationKey": "DASHBOARD_TITLE_SOCIAL_ECONOMIC_KEY",
+        "type": "vitals",
+        "isObservation": true,
+        "displayOrder": 13,
+        "dashboardConfig": {
+          "showDetailsButton": true,
+          "conceptNames": [
+            "Profession",
```
Comment (rbuisson) [inline configuration/openmrs/apps/clinical/dashboard.json:271]: Is this the right concept? In english, shouldn't this be Occupation?

File: configuration/openmrs/obscalculator/BahmniObsValueCalculator.groovy [hunk 2]
```diff
@@ -55,26 +58,47 @@ public class BahmniObsValueCalculator implements ObsValueCalculator {
         Collection<BahmniObservation> observations = bahmniEncounterTransaction.getObservations()
         def nowAsOfEncounter = bahmniEncounterTransaction.getEncounterDateTime() != null ? bahmniEncounterTransaction.getEncounterDateTime() : new Date();
 
-        BahmniObservation heightObservation = find("Height", observations, null)
-        BahmniObservation weightObservation = find("Weight", observations, null)
+ BahmniObservation heightObservation = findByUUID("5090AAAAAAAAAAAAAAAAAAAAAAAAAAAA", observations, null);
```
Comment (rbuisson) [inline configuration/openmrs/obscalculator/BahmniObsValueCalculator.groovy]: Could you specify this as a constant at the top of the file? and substitute any further references by the constant.

File: configuration/openmrs/obscalculator/BahmniObsValueCalculator.groovy [hunk 3]
```diff
@@ -99,8 +123,13 @@ public class BahmniObsValueCalculator implements ObsValueCalculator {
...
 
-            bmiDataObservation = bmiDataObservation ?: createObs("BMI Data", parent, bahmniEncounterTransaction, obsDatetime) as BahmniObservation
-            bmiStatusDataObservation = bmiStatusDataObservation ?: createObs("BMI Status Data", parent, bahmniEncounterTransaction, obsDatetime) as BahmniObservation
+            if(encounter != null) {
+                voidPreviousBMIObs(encounter.getObsAtTopLevel(false));
+                voidPreviousBMIObs(encounter.getObs());
+            }
+
+            bmiDataObservation = bmiDataObservation ?: createObs("BMI Data", null, bahmniEncounterTransaction, obsDatetime) as BahmniObservation
+            bmiStatusDataObservation = bmiStatusDataObservation ?: createObs("BMI Status Data", null, bahmniEncounterTransaction, obsDatetime) as BahmniObservation
```
Comment (rbuisson) [inline configuration/openmrs/obscalculator/BahmniObsValueCalculator.groovy:135]: How does that play in other locales?
Should we use UUID here as well?

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/openmrs/obscalculator/BahmniObsValueCalculator.groovy (hunk 3 above)
Comment (reagan-meant) [reply configuration/openmrs/obscalculator/BahmniObsValueCalculator.groovy:135]: We are using the same name in all locale FSNs.

File: configuration/openmrs/obscalculator/BahmniObsValueCalculator.groovy (hunk 2 above)
Comment (reagan-meant) [reply configuration/openmrs/obscalculator/BahmniObsValueCalculator.groovy]: Done

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 1 above)
Comment (reagan-meant) [reply configuration/openmrs/apps/clinical/dashboard.json:271]: It is fine in English except for the sake of Haiti-Distro alignment we can make it Occupation

**Pull request merged**